webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -244,52 +245,40 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
   const [error] = useState(false);
   const [canceler] = useState(new AbortController());
 
-  // partition experiment list into not selected/selected with indices and experiments so we only iterate the result list once
-  const [excludedExperimentIds, selectedExperimentIds] = useMemo(() => {
+  const loadedSelectedExperimentIds = useMemo(() => {
     const selectedMap = new Map<number, ExperimentWithIndex>();
-    const excludedMap = new Map<number, ExperimentWithIndex>();
     if (isLoadingSettings) {
-      return [excludedMap, selectedMap];
+      return selectedMap;
     }
     const selectedIdSet = new Set(
       settings.selection.type === 'ONLY_IN' ? settings.selection.selections : [],
     );
-    const excludedIdSet = new Set(
-      settings.selection.type === 'ALL_EXCEPT' ? settings.selection.exclusions : [],
-    );
     experiments.forEach((e, index) => {
       Loadable.forEach(e, ({ experiment }) => {
-        const mapToAdd =
-          (selectAll && !excludedIdSet.has(experiment.id)) || selectedIdSet.has(experiment.id)
-            ? selectedMap
-            : excludedMap;
-        mapToAdd.set(experiment.id, { experiment, index });
+        if (selectedIdSet.has(experiment.id)) {
+          selectedMap.set(experiment.id, { experiment, index });
+        }
       });
     });
-    return [excludedMap, selectedMap];
-  }, [isLoadingSettings, selectAll, settings.selection, experiments]);
+    return selectedMap;
+  }, [isLoadingSettings, settings.selection, experiments]);
+
+  const allSelectedExperimentIds = useMemo(() => {
+    return settings.selection.type === 'ONLY_IN' ? settings.selection.selections : [];
+  }, [settings.selection]);
```

Comment:
move this up to be a dependency of `LloadedSelectedExperimentIds`

Reply:
fixed